src/devices_models/devices/electric_loads.jl

```diff
@@ -4,44 +4,31 @@ struct StaticPowerLoad <: AbstractLoadFormulation end
 struct InterruptiblePowerLoad <: AbstractControllablePowerLoadFormulation end
 struct DispatchablePowerLoad <: AbstractControllablePowerLoadFormulation end
 
-########################### dispatchable load variables ####################################
-function AddVariableSpec(
-    ::Type{T},
-    ::Type{U},
-    ::PSIContainer,
-) where {T <: ActivePowerVariable, U <: PSY.ElectricLoad}
-    return AddVariableSpec(;
-        variable_name = make_variable_name(T, U),
-        binary = false,
-        expression_name = :nodal_balance_active,
-        sign = -1.0,
-        lb_value_func = x -> 0.0,
-        ub_value_func = x -> PSY.get_max_active_power(x),
-    )
-end
+########################### ElectricLoad ####################################
 
-function AddVariableSpec(
-    ::Type{T},
-    ::Type{U},
-    ::PSIContainer,
-) where {T <: ReactivePowerVariable, U <: PSY.ElectricLoad}
-    return AddVariableSpec(;
-        variable_name = make_variable_name(T, U),
-        binary = false,
-        expression_name = :nodal_balance_reactive,
-        sign = -1.0,
-        lb_value_func = x -> 0.0,
-        ub_value_func = x -> PSY.get_max_reactive_power(x),
-    )
-end
+get_variable_sign(_, ::Type{<:PSY.ElectricLoad}) = -1.0
 
-function AddVariableSpec(
-    ::Type{T},
-    ::Type{U},
-    ::PSIContainer,
-) where {T <: OnVariable, U <: PSY.ElectricLoad}
-    return AddVariableSpec(; variable_name = make_variable_name(T, U), binary = true)
-end
+########################### ActivePowerVariable, ElectricLoad ####################################
+
+get_variable_binary(::ActivePowerVariable, ::Type{<:PSY.ElectricLoad}) = false
+
+get_variable_expression_name(::ActivePowerVariable, ::Type{<:PSY.ElectricLoad}) = :nodal_balance_active
```

Comment:
This part could be problematic in the future as a variable could be added to different expressions. I think we talked about it before. For now is ok as nothing is breaking but this could require making It a function of the formulation too.

Reply:
We can extend it to support future use cases.